.github/workflows/ci.yml

```diff
@@ -25,13 +25,20 @@ jobs:
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
         python-version: [3.9]
+        majorminor: ["3.9"]
```

Comment:
A bit tedious but it should work?

This also opens the door to easily testing other Python distributions like PyPy, for which we'll have the same issue because the GA setup-python name won't match the nox major.minor python version session spec..